### PR TITLE
fix singleton error.

### PR DESCRIPTION
**Pull request by zxcfortest**
Thank you for your contribution to the Sandboxie repository.

In order to reduce the risk of accidental merges, it's highly recommended for beginners to open a new Pull Request in draft state by clicking the button "Create Draft Pull Request", instead of using the default "Create Pull Request". You can convert an existing pull request to a draft by clicking the "Convert to draft" link in the right sidebar under "Reviewers".

### CODE REVIEW

File: Sandboxie/common/my_version.h [hunk 1]
```diff
@@ -32,8 +32,8 @@
 
 #define MY_PRODUCT_NAME_STRING  "Sandboxie"
 #define MY_COMPANY_NAME_STRING  "sandboxie-plus.com"
-#define MY_COPYRIGHT_STRING     "Copyright � 2020-2021 by David Xanatos (xanasoft.com)"
-#define MY_COPYRIGHT_STRING_OLD "Copyright � 2004-2020 by Sandboxie Holdings, LLC"
+#define MY_COPYRIGHT_STRING     "Copyright ?2020-2021 by David Xanatos (xanasoft.com)"
+#define MY_COPYRIGHT_STRING_OLD "Copyright ?2004-2020 by Sandboxie Holdings, LLC"
```
Comment (isaak654) [inline Sandboxie/common/my_version.h:36]: Why the question marks?

**Comment by DavidXanatos** [comment]
the original code seams to work just fine, what issue is you change resolving?
also plrease revert the formating change on the version.h file as we need the unicode copyright charakter

**Comment by isaak654** [comment]
I removed the format change that was causing the failed build message.

Now I would just need a second opinion about the opportunity to merge this.

**Comment by isaak654** [comment]
@Flamencist just confirmed me that the existed logic is fine.
No need to change OpenMutex to CreateMutex and checking of last error.

**Pull request closed without merging**